Source/Classes/Drawing/Drawing.swift

```diff
@@ -118,7 +118,7 @@ extension NantesLabel {
             CTLineGetTypographicBounds(line, nil, &descent, nil)
 
             let penOffset = CGFloat(CTLineGetPenOffsetForFlush(line, flushFactor, Double(rect.size.width)))
-            let yOffset = lineOrigin.y - descent - font.descender
+            let yOffset = lineOrigin.y
```

Comment:
This is the code that causes the bug. UIFont.descender is always negative value. `descent` is a Core Text value that  does gives similar value with UIFont.descender. This cause it renders at higher position than expected.

This calculate isn't necessary

Reply:
can we remove the `descent` variable and the `CTLineGetTypographicBounds` call then?

Reply:
yeah! right, will do. thanks!